tests/FakeItEasy.Tests.Approval/ApiApproval.cs

```diff
@@ -1,79 +1,40 @@
 namespace FakeItEasy.Tests.Approval
 {
     using System;
-    using System.Diagnostics.CodeAnalysis;
     using System.IO;
     using System.Reflection;
     using System.Runtime.CompilerServices;
     using ApprovalTests;
+    using ApprovalTests.Core;
     using ApprovalTests.Reporters;
+    using ApprovalTests.Writers;
     using PublicApiGenerator;
     using Xunit;
 
     public class ApiApproval
     {
-        [Fact]
+        [Theory]
+        [InlineData("net40")]
+        [InlineData("net45")]
+        [InlineData("netstandard1.6")]
+        [InlineData("netstandard2.0")]
+        [InlineData("netstandard2.1")]
         [UseReporter(typeof(DiffReporter))]
         [MethodImpl(MethodImplOptions.NoInlining)]
-        public void ApproveApi45()
+        public void ApproveFakeItEasyApi(string frameworkVersion)

```

Comment:
When I started reading, I was expecting to see a test parameterized by project name and frameworkVersion. This is good, though!

Reply:
I pushed a new commit implementing that change